config/default.yml

```diff
@@ -907,6 +907,12 @@ RSpec/Rails/AvoidSetupHook:
   VersionAdded: '2.4'
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/Rails/AvoidSetupHook
 
+RSpec/Rails/HaveHttpStatus:
+  Description: Checks that tests use `have_http_status` instead of equality matchers.
+  Enabled: pending
+  VersionAdded: '2.12'
```

Comment:
It may happen that the release will be 2.13. 2.12 is about to happen https://github.com/rubocop/rubocop-rspec/pull/1309

Reply:
Can you please add `'<<next>>'` for the version? We'll replace it with a version during the release.

Reply:
Fixed in https://github.com/rubocop/rubocop-rspec/pull/1307/commits/cf65006851b20c85b88b2b6ab84e015866eed351